gaia-header.js

```diff
@@ -91,6 +91,11 @@ module.exports = component.register('gaia-header', {
       titles: this.getElementsByTagName('h1')
     };
 
+    // The [ignoreDir] attribute can be used to force the older behavior of
+    // gaia-header, where the whole header is always displayed in LTR mode.
+    this.ignoreDir = this.hasAttribute('ignoreDir');
```

Comment:
nit: move this closer to the other `getAttribute` calls ? I understand this needs to be set before setting any other attribute though.

Also wondering about adding it to the "attrs" dictionary, to be able to react to any change from outside.. but I think it's likely not necessary as we won't change it dynamically.